### DISCUSSION TITLE
右侧分区导航中的分区锚点跟随首页过滤设置隐藏

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1448] [功能建议 / Ideas]
右侧分区导航中的分区锚点跟随首页过滤设置隐藏。
隔壁 https://github.com/bilibili-helper/bilibili-helper-o 就会跟着隐藏。

**Comment by the1812** [comment]
我也想, 但是没有什么办法确定藏哪个锚点, 按文字的话开界面翻译就不行了

**Reply by Geeyun-JY3** [reply]
功能太多的烦恼？（

**Comment by Geeyun-JY3** [comment]
旧版首页大概已经回不去了……
